src/game-app.js

```diff
@@ -340,6 +340,10 @@ var buttons = {
     $('#game').toggleClass('is-busy');
     return false;
   },
+  fork: async () => {
+    let gameId = game.state._data.id;
```

Comment:
This should be `const` not `let`.

Reply:
I'd rather remove this altogether, which I did